src/forms/validators/default-value-validator.ts

```diff
@@ -0,0 +1,25 @@
+import moment from 'moment';
+import { FieldValidator, OHRIFormField } from '../types';
+
+export const OHRIDefaultFieldValueValidator: FieldValidator = {
```

Comment:
Not sure if that's possible in react/js but in old school languages what I would use the registry pattern, I would create methods like
`checkDateValue` , `checkListTypeFieldValue` (for radio, checkbox, ..),`checkNumberValue` etc and I would register them against the fieldTypes then here I would just call them based on the file type. 

I REALLY DON'T LIKE IF's